### PR TITLE
Adds new flows and expands ReCiPe mapping

### PR DESCRIPTION
**Pull request by bl-young**

### CODE REVIEW

**Review by bnjmnmorelli — APPROVED** [review]
ignore this. placed comment in wrong spot and can't delete.

**Review by bnjmnmorelli — COMMENTED** [review]
@bl-young I am not sure I understand this addition. If you look at the top 8 rows (for example) I understand why Ag (I) air/rural is being mapped to multiple FEDEFL contexts. But why is Ag (I) also being mapped to multiple target flows (Silver (I) and Silver)?

**Comment by bl-young** [comment]
> @bl-young I am not sure I understand this addition. If you look at the top 8 rows (for example) I understand why Ag (I) air/rural is being mapped to multiple FEDEFL contexts. But why is Ag (I) also being mapped to multiple target flows (Silver (I) and Silver)?

This was added in https://github.com/USEPA/fedelemflowlist/pull/167, which has some discussion there. These LCIA mappings are reversed, in that we can have two flows map to the same CF, if desired. In this case, we are saying that both `Silver` and `Silver(I)` should use the ReCiPe CF denoted by Ag(I). Otherwise, as it stands prior to this change, emissions of silver to air are not characterized.

File: fedelemflowlist/flowmapping/ReCiPe2016.csv [hunk 1]
```diff
@@ -168671,6 +168671,7 @@ ReCiPe2016,m-Xylene,,water/unspecified,kg,=,1.0,m-Xylene,8e51082a-704d-382f-bb6a
 ReCiPe2016,m-Xylene,,water/unspecified,kg,=,1.0,m-Xylene,3bd0d9ec-9c4e-38b8-888c-d00408f5ade5,emission/water/subterranean/brackish water body/confined aquifer,kg,Hottle,Edelen,1/7/2020
 ReCiPe2016,m-Xylene,,water/unspecified,kg,=,1.0,m-Xylene,06b717e8-b9bf-3a21-992a-20c7b010f769,emission/water/subterranean/brackish water body/unconfined aquifer,kg,Hottle,Edelen,1/7/2020
 ReCiPe2016,managed forest,,managed forest,m2*a,=,1.0,Land use,1147e66c-db3e-303c-b7c7-89b006d98881,resource/ground/terrestrial/forest,m2*a,Hottle,,2/25/2020
+ReCiPe2016,managed forest,,managed forest,m2*a,~,1.0,Land use,5d89617f-b66c-3e43-ac10-7f322a4ca29b,resource/ground/terrestrial/shrubland,m2*a,Filipe,Daniel,6-20-2023
```
Comment (bnjmnmorelli) [inline fedelemflowlist/flowmapping/ReCiPe2016.csv]: i don't understand this mapping. the row above seems more appropriate. If the goal is to map all FEDEFL contexts to a recipe flow then we are still missing contexts such as 'Land use', 'resource/ground/terrestrial/barren land'.

Comment (bl-young) [reply fedelemflowlist/flowmapping/ReCiPe2016.csv]: Justification for that one is here: https://github.com/USEPA/fedelemflowlist/pull/166

**Review by bnjmnmorelli — APPROVED** [review]
looks good apart from specific comments.

File: fedelemflowlist/input/ChemicalsFlowablePrimaryContexts.csv [hunk 2]
```diff
@@ -15003,3 +15015,70 @@ Allyl 2-hydroxyisobutyrate,Emission,Ground
...
+Octaethylene glycol monododecyl ether,Emission,Air
+2-[4-[2-hydroxy-3-(propan-2-ylamino)propoxy]phenyl]acetamide,Emission,Ground
+2-[4-[2-hydroxy-3-(propan-2-ylamino)propoxy]phenyl]acetamide,Emission,Water
+"(3R,5R)-7-[2-(4-fluorophenyl)-3-phenyl-4-(phenylcarbamoyl)-5-propan-2-ylpyrrol-1-yl]-3,5-dihydroxyheptanoic acid",Emission,Water
+"(3R,5R)-7-[2-(4-fluorophenyl)-3-phenyl-4-(phenylcarbamoyl)-5-propan-2-ylpyrrol-1-yl]-3,5-dihydroxyheptanoic acid",Emission,Ground
+"1-(6-tert-butyl-1,1-dimethyl-2,3-dihydroinden-4-yl)ethanone",Emission,Ground
+"1-(6-tert-butyl-1,1-dimethyl-2,3-dihydroinden-4-yl)ethanone",Emission,Water
+2-(4-chlorophenoxy)-2-methylpropanoic acid,Emission,Water
+2-(4-chlorophenoxy)-2-methylpropanoic acid,Emission,Ground
+"2,2-dichloroacetaldehyde",Emission,Water
```
[the remark this answers is not in the document]
Comment (bl-young) [reply fedelemflowlist/input/ChemicalsFlowablePrimaryContexts.csv:15060]: if there are commas within terms it will automatically add quotation marks

**Pull request merged**